### PR TITLE
Add recipe for yapsy

### PR DESCRIPTION
**Pull request by ganeshhubale**
Related to #4232

### CODE REVIEW

**Comment by ganeshhubale** [comment]
The package doesn't contain a LICENSE file.

File: recipes/yapsy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "yapsy" %}
```
Comment (synapticarbors) [inline recipes/yapsy/meta.yaml]: You need to change this to `Yapsy` (note capitalization):

https://pypi.org/project/Yapsy/#files

File: recipes/yapsy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
```
Comment (synapticarbors) [inline recipes/yapsy/meta.yaml]: consider making `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

File: recipes/yapsy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - yapsy
+
+about:
+  home: http://yapsy.sourceforge.net
+  license: BSD
```
Comment (synapticarbors) [inline recipes/yapsy/meta.yaml]: `license: BSD-2-Clause`

File: recipes/yapsy/meta.yaml (hunk 2 above)
Comment (ganeshhubale) [reply recipes/yapsy/meta.yaml]: Thank you

**Comment by synapticarbors** [comment]
@ganeshhubale -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Comment by ganeshhubale** [comment]
I have just started my journey in open source world. It will be great inspiration for me. 
Thank you,

**Comment by jakirkham** [comment]
That's awesome @ganeshhubale! Welcome. 😄

Also thanks for adding `yapsy`.

**Comment by jakirkham** [comment]
cc @stuarteberg

**Comment by stuarteberg** [comment]
cc @ilastik

**Pull request merged**